app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/ColorCorrect.scala

```diff
@@ -153,25 +153,65 @@ object ColorCorrect {
     val maybeEqualize =
       if (params.equalize) Some(HistogramEqualization(_: MultibandTile, rgbHist)) else None
 
+    case class LayerClipping(redMin: Int, redMax: Int, greenMin: Int, greenMax:Int, blueMin: Int, blueMax: Int)
+    case class ClippingParams(band: Int, min: Option[Int], max: Option[Int], layerMin: Int, layerMax: Int)
```

Comment:
It seems like this class is being used in two different modes: either the `min` and `max` fields are both `None` or the `layerMin` and `layerMax` are `0` and `255`, respectively. Does it make sense to split this into two separate classes that directly map to the two usage modes, or perhaps generalize it to just hold a min and a max?